README.md

```diff
@@ -19,25 +53,187 @@ docker-compose run --rm bety initialize
 docker-compose run --rm bety sync
 ```
 
+### Running the App
 
-## Running the App
+Once you have completed this step, YABA interface will be running and you can upload your Metadata at [localhost:3001](http://localhost:3001/)
 
-### Bring up container for YABA-API
+#### Bring up the containers at once
+
+```sh
+docker-compose up
+```
+
+#### Bring up container for YABA-API
+
+If you only want to run the yaba-api, not the interface and yaba-client, you can do this.
 
 ```sh
 docker-compose up -d yaba_api
 ```
 
-## Running the App (without initialization and synchronization of bety)
+### Running the App without initialization and synchronization of bety
+
+Once you have completed this step, YABA interface will be running and you can upload your Metadata at [localhost:3001](http://localhost:3001/)
 
-### Bring up the containers at once
+#### Skip the "Setup BETYdb" step and bring up the containers at once
 
-#### Once initialization and synchronization of bety database is done,the below command can be directly used to start the app.(Skip the "Setup BETYdb" step) 
 ```sh
 # Bring up full stack
 docker-compose up
 ```
-## How to hit the YABA API endpoints
+
+## Steps to Upload Metadata
+
+**1.** Enter your API key
```

Comment:
If just the numbers with the following period are here, instead of bolding them, it will make it into an actual list. [Markdown list syntax](https://www.markdownguide.org/basic-syntax/#ordered-lists). I showed how to do this for the first list item below: 

```suggestion
1. Enter your API key
```

Reply:
Changed in future commit